app/coffee/Features/Collaborators/CollaboratorsInviteController.coffee

```diff
@@ -0,0 +1,146 @@
+ProjectGetter = require "../Project/ProjectGetter"
+LimitationsManager = require "../Subscription/LimitationsManager"
+UserGetter = require "../User/UserGetter"
+CollaboratorsHandler = require('./CollaboratorsHandler')
+CollaboratorsInviteHandler = require('./CollaboratorsInviteHandler')
+logger = require('logger-sharelatex')
+EmailHelper = require "../Helpers/EmailHelper"
+EditorRealTimeController = require("../Editor/EditorRealTimeController")
+NotificationsBuilder = require("../Notifications/NotificationsBuilder")
+
+
+module.exports = CollaboratorsInviteController =
+
+	getAllInvites: (req, res, next) ->
+		projectId = req.params.Project_id
+		logger.log {projectId}, "getting all active invites for project"
+		CollaboratorsInviteHandler.getAllInvites projectId, (err, invites) ->
+			if err?
+				logger.err {projectId}, "error getting invites for project"
+				return next(err)
+			res.json({invites: invites})
+
+	_trySendInviteNotification: (projectId, sendingUser, invite, callback=(err)->) ->
```

Comment:
this feels like it should live in the handler not controller. there should ideally be 1 function to call which actually sends the email and notification.